src/client/components/AchievementComponent/AchieveComponent.styles.css

```diff
@@ -0,0 +1,40 @@
+@import url('https://fonts.googleapis.com/css2?family=Rammetto+One&family=Roboto&display=swap');
+
+.achieveMain {
+  display: grid;
+  grid-template-columns: repeat(2, 0fr);
+  grid-template-rows: repeat(2, 13vw);
+  grid-gap: 10px;
+}
+.AchieveBox {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  width: 155px;
+  height: 124px;
+  margin-top: 25px;
+  margin-left: 10px;
+  background: #fba85c;
+  box-shadow: 0px 0px 15px rgba(0, 0, 0, 0.09);
+  border-radius: 5px;
+}
+.AchieveCircle {
+  display: flex;
+  justify-content: center;
+  width: 60px;
+  height: 60px;
+  margin-top: -20px;
+  background: #e3e3e9;
+  border-radius: 50%;
+  border-style: solid;
+  border-width: 1px;
+  border-color: darkgrey;
+}
+img {
+  width: 30px;
```

Comment:
We should never fix the dimensions of an image, it should always be 100% inheriting the size of it's parent containers. Use AchieveCircle class to do so. put some padding and it will be set without fixed width 🙂

Reply:
Ohhh! I never tried this before :) it's also working without put code for "img" class in CSS file, so we reduced the numbers of codes :)

Reply:
Ohhh! I never tried this before :) it's also working without put code for "img" class in CSS file, so we reduced the numbers of codes :)

Reply:
Exactly my point 🙂